example.js

```diff
@@ -0,0 +1,25 @@
+
+var Stagger = require('./lib/stagger');
+
+var stagger = Stagger.getDefault({
+  "pid": "1234",
+  "cmd": "./hello"
```

Comment:
So the library should set all the tags? Should there be a way of the client application to set these too?